help/destinations/catalog/mobile-engagement/line.md

```diff
@@ -0,0 +1,141 @@
+---
+keywords: mobile;mobile engagement destinations;LINE;LINE mobile engagement destination
+title: LINE connection
+description: The LINE destination allows you to add profiles to your Platform segment and deliver personalized experiences to connected users.
+---
+
+# [!DNL LINE] connection
+
+## Overview {#overview}
+
+[[!DNL LINE]](https://line.me/en/) is a popular communication platform that connects people, services and information and has grown from a chat app into a hub for entertainment, social and day-to-day activities.
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL LINE] Messaging API](https://developers.line.biz/en/reference/messaging-api/), add profiles to your segment and activate them as connections within [!DNL LINE] for your business needs.
+
+[!DNL LINE] uses Bearer Tokens as the authentication mechanism to communicate with the [[!DNL LINE] Messaging API]. Instructions to authenticate to your [!DNL LINE] instance are further below, within [Authenticate to destination](#authenticate) section.
+
+## Use cases {#use-cases}
+
+As a marketer, I want to target users in a mobile engagement destination, with segments built in [!DNL Adobe Experience Platform]. Additionally, I want to deliver personalized experiences to them, based on attributes from their [!DNL Adobe Experience Platform] profiles, as soon as segments and profiles are updated in [!DNL Adobe Experience Platform].
+
+## Prerequisites {#prerequisites}
+
+### Experience Platform prerequisites {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL LINE] destination, you must have a [schema](/help/xdm/schema/composition.md), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+Read [Activate profiles and segments to streaming segment export destinations](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/activate/activate-segment-streaming-destinations.html?lang=en) for instructions on activating audience segments to this destination.
+
+### LINE prerequisites {#prerequisites-destination}
```

Comment:
Let's be consistent and use [!DNL LINE] throughout the doc. I see it is missing here and in the two `####` headings below.